docs/csharp/language-reference/language-specification.md

```diff
@@ -34,8 +34,10 @@ translation.priority.ht:
 # C# Language Specification
 The C# Language Specification is the definitive source for C# syntax and usage. This spec contains detailed information about all aspects of the language, including many points that the documentation for Visual C# doesn't cover.  
   
- You can download this spec from the [Microsoft Download Center](http://www.microsoft.com/download/details.aspx?id=7029). If you've installed Visual Studio 2013, you can also find the spec on your computer in the Program Files (x86)/Microsoft Visual Studio 12.0/VC#/Specifications/1033 folder. However, installations of Visual Studio Express 2013 don't include this file.  
+You can download version 5.0 of this spec from the [Microsoft Download Center](http://www.microsoft.com/download/details.aspx?id=7029). If you've installed Visual Studio 2015, you can also find the spec on your computer in the Program Files (x86)/Microsoft Visual Studio 14.0/VC#/Specifications/1033 folder. However, installations of Visual Studio Express 2015 don't include this file.
```

Comment:
I finally managed to verify this. I've installed VS 2015 in Italian in a VM. The path definitely varies by locale. The spec was under C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC#\Specifications\1040

But the most surprising thing to me was to learn that the spec was also translated:
![image](https://cloud.githubusercontent.com/assets/12971179/23526967/e0a519da-ff48-11e6-9487-24c795b878a4.png)